apps/docs/src/guide/tooling/graphql-integration.md

```diff
@@ -0,0 +1,16 @@
+# GraphQL Integration
+
+The Fuel Network provides a GraphQL API to query the blockchain ([docs](https://docs.fuel.network/docs/graphql/overview/)).
+
+For its own purposes, the SDK creates custom operations based off of the API's schema and auto-generates TypeScript client code via codegen tools.
+The details of our integration can be found in the source code in the relevant [scripts](https://github.com/FuelLabs/fuels-ts/blob/e6df29c2d4ef373c6d266ba08110d6480732f0e1/packages/account/package.json#L42) that automate the process.
```

Comment:
I don't think we should be telling people to look through source code to understand the process, we should explain that process here. Or potentially we should just remove this section.

Reply:
Then I'd rather remove it because anyone who wants to use codegen tools can google how to set it up.